benchmarking/definitions/definition_nashpobench.py

```diff
@@ -71,7 +67,6 @@ def nashpobench_benchmark(params):
         'cost_model': None,
         'supports_simulated': True,
         'blackbox_name': BLACKBOX_NAME,
-        'surrogate': RandomForestRegressor(max_samples=0.005, bootstrap=True),
+        'surrogate': None,
```

Comment:
Could you revert this line? (it will not work currently)

Reply:
What do you mean? This idea here is not to have a surrogate. I removed the whole line.

Reply:
BTW: When I am accessing an entry in a blackbox table, where a HP has float as type, do I need to hit the float value in the table exactly? In principle, finrange and logfinrange with value type float could not hit the intended values exactly. The cast_int is a mechanism to keep things safe for int values, which is good.

Reply:
> What do you mean? This idea here is not to have a surrogate. I removed the whole line.

Indeed you are right.

Regarding your question, yes the value must be exact otherwise a surrogate model must be used.